pandas/_libs/tslibs/conversion.pyx

```diff
@@ -664,7 +670,8 @@ cdef inline int64_t[:] _tz_convert_dst(int64_t[:] values, tzinfo tz,
 
 
 cdef inline int64_t _tz_convert_tzlocal_utc(int64_t val, tzinfo tz,
-                                            bint to_utc=True):
+                                            bint to_utc=True,
+                                            bint from_utc=False):
```

Comment:
Isn't `from_utc` just equal to `not to_utc`?

Reply:
Hmm it appears that way in general. I can try to remove the new keyword arg for `not to_utc`